### PR TITLE
Add notes re dangers of identity write endpoints.

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: website/content/api-docs/secret/identity/entity-alias.mdx [hunk 1]
```diff
@@ -6,6 +6,10 @@ description: >-
   store.
 ---
 
+~> **NOTE:** Be careful in granting permissions to non-readonly identity group
+endpoints. If they can modify group membership, they can add their entity to
```
Comment (mladlow) [inline website/content/api-docs/secret/identity/entity-alias.mdx:10]: You use "If a user" elsewhere and I think that reads more clearly.

```suggestion
endpoints. If a user can modify group membership, they can add their entity to
```

File: website/content/api-docs/secret/identity/entity-alias.mdx [hunk 2]
```diff
@@ -6,6 +6,10 @@ description: >-
   store.
 ---
 
+~> **NOTE:** Be careful in granting permissions to non-readonly identity group
```
Comment (mladlow) [inline website/content/api-docs/secret/identity/entity-alias.mdx:9]: I couldn't tell what the word "group" meant here. Should this sentence go on the group page and be switched with the one that's currently on the group page?

File: website/content/api-docs/secret/identity/group.mdx [hunk 3]
```diff
@@ -4,6 +4,11 @@ page_title: 'Identity Secret Backend: Group - HTTP API'
 description: This is the API documentation for managing groups in the identity store.
 ---
 
+~> **NOTE:** Be careful in granting permissions to non-readonly identity endpoints.
+If a user can modify an entity, they can grant it additional privileges through
+policies.  If they can modify an alias they can login with, they can bind it to
+an entity with higher privileges.
```
Comment (mladlow) [inline website/content/api-docs/secret/identity/group.mdx:10]: Maybe?
```suggestion
an entity with higher privileges as long as they have access to the entity with more privileges.
```

**Review by mladlow — APPROVED** [review]
I have a couple suggestions, and it looks like the note on the identity/group page should be switched with the entity-alias page?

File: website/content/api-docs/secret/identity/group.mdx (hunk 3 above)
Comment (ncabatoff) [reply website/content/api-docs/secret/identity/group.mdx:10]: Not sure what you're getting at here?

File: website/content/api-docs/secret/identity/entity-alias.mdx (hunk 2 above)
Comment (ncabatoff) [reply website/content/api-docs/secret/identity/entity-alias.mdx:9]: Yeah, got it backwards.

File: website/content/api-docs/secret/identity/group.mdx (hunk 3 above)
Comment (mladlow) [reply website/content/api-docs/secret/identity/group.mdx:10]: The first time I read it my initial reading included some "they can bind it to ANY entity with higher privileges" and not "they can bind it to an entity they already have access to". This suggestion definitely gets into areas where my understanding is kind of hazy though, so I don't mind if you deem it nonsensical.

Comment (ncabatoff) [reply website/content/api-docs/secret/identity/group.mdx:10]: It is the former, i.e. ANY entity.  That's the motivation for this warning.

**Pull request merged**